_gsocprojects/2020/project_CMSSW.md

```diff
@@ -0,0 +1,12 @@
+---
+project: CMSSW
+layout: default
+description: |
+  The LHC smashes groups of protons together at close to the speed of light: 40 million times per second and with seven times the energy of the most powerful accelerators built up to now. Many of these will just be glancing blows but some will be head on collisions and very energetic. When this happens some of the energy of the collision is turned into mass and previously unobserved, short-lived particles – which could give clues about how Nature behaves at a fundamental level - fly out and into the detector. Our work includes the experimental discovery of the Higgs boson, which lead to the award of a Nobel prize for the underlying theory that predicted the Higgs boson as an important piece of the standard model theory of particle physics.
```

Comment:
First question: do you want to make this a standalone project or connect its proposals to the main CMS project that is already published on our page? Event If you want to keep it as standalone project, you should cut away a large part of the first and second paragraphs that describe CMS without referring to CMSSW, to minimize the clash/confusion with the existing CMS project. The text is anyway not well adapted for the short description that should appear in the main page. I guess the intent was to have the following paragraph as summary, but the syntax is wrong - have a look at other projects in the same directory. If you decide to have CMSSW parented to the CMS project, you can completely drop this file and make a short description of CMSSW in each of the related proposals.

Reply:
I think we should have a separate project for CMSSW. Both ways may be confusing. I was confused by the current state of art because we have the only submitted proposal for CMS is not in the context of CMSSW. I will reword.